tests/test_event_hooks.py

```diff
@@ -0,0 +1,81 @@
+from httpx._config import EventHooks
+
+
+def callable():
+    pass  # pragma: nocover
+
+
+def test_event_hooks_init():
```

Comment:
These tests would be simplified if we adopted a simple `create_event_hooks()` helper that returns a plain dict.